docs/evolutionaryAlgorithm/README.md

```diff
@@ -0,0 +1,269 @@
+
+```
+? Sorting direction of cost results (Use arrow keys)
+❯ Maximize (higher value is better) 
+  Minimize (lower value is better) 
+```
+
+Fitness functions for the simulation can then be selected. One or many can be used. If many functions are used, a weight can be given to each method and the final fitness is calculated as follows, from a comment in the code: `TODO Figure out how to get the totalFitness of a candidate given the results. For now, we use an approach inspired by the climbing Olympic discipline: the totalFitness is the product of the rank of each candidate for each method. The rank of each candidate for each method is defined as the power of the actual rank order and the weight of the method`.
+
+```
+? Select functions to use for simulation (Press <space> to select, <a> to toggle all, <i> to invert 
+selection, and <enter> to proceed)
+❯◯ OD trips
+ ◯ Using accessibility maps
+? Weight of the cost for simulation function "OD trips" 0.5
+```
+
+Then, specific parameters for each methods are asked. 
+
+For the `OD trips` method, it asks the data source to use, as well as the sample ratio to use (for large od trips data source, only a random fraction of the trips can be used to simulate). A ratio of 1 means the whole data source will be used.
+
+Then it asks for the cost function to use to determine the cost of a single od trip result. It can either use the travel time only in the function, or add an extra penalty for transfers.
+
+*Note*: These cost functions are defined in the `defaultPreferences.config.ts` file and should be better documented. For example, the travelTime is multiplied by 10 in the function. 10$/minute? Where does that number come from? And why not penalize with the same cost the initial lost time? Also, the text of the second method says 10 min penalty for transfers, but it seems to be actually 5?
+
+Then, you need to select the cost function for the entire scenario (the fitness used to rank the various chromosomes). It can either be a function of the user cost (as defined by the odTrip cost function?) or the operation cost (to run the fleet for this scenario), or both.
+
+*Note*: These cost functions are defined in the `defaultPreferences.config.ts` file and should be better documented. I, tahini, don't know what else to say here...
+
+```
+? Data source for OD (Use arrow keys)
+❯ odTrips
+? Sample ratio (between 0 and 1)
+? Cost function for OD trip result (Use arrow keys)
+❯ Travel time
+  Travel time + 10 min penalty for transfers
+? Cost function for the scenario (Use arrow keys)
+❯ User + operating costs (hourly)
+  User cost (hourly)
+  Operating cost (hourly)
+```
+
+For the `Accessibility maps` method, it asks for the data source to use and the maximum sample ratio to use from that datasource. By default, it will stop after a while, when the result variability is very low, but as long as variability changes, it will test places until that ratio is reached.
```

Comment:
Could you explain a bit more here, I'm not following

Reply:
I changed the text, tell me if it's clearer now

Reply:
Ok, much better I get it now